consul/client.go

```diff
@@ -150,6 +151,10 @@ func (c *Client) setupSerf(conf *serf.Config, ch chan serf.Event, path string) (
 	conf.Tags["vsn_min"] = fmt.Sprintf("%d", ProtocolVersionMin)
 	conf.Tags["vsn_max"] = fmt.Sprintf("%d", ProtocolVersionMax)
 	conf.Tags["build"] = c.config.Build
+	conf.Tags["rpc_port"] = fmt.Sprintf("%d", c.config.RPCAddr.Port)
```

Comment:
I think that's right, I thought the keyring or event stuff might use it, but those both use serf to gossip out, so we can remove this